plugins/jobs/src/js/JobsToggleSchedule.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import { graphqlObservable } from "data-service";
+import gql from "graphql-tag";
+
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/do";
+import "rxjs/observable/empty";
+
+import { jobsMenuQLSchema } from "./JobsMenuQLSchema";
+
+const enableJobMutation = gql`
+  mutation {
+    updateSchedule(jobId: $jobId, data: $data) {
```

Comment:
This is a js template string, that will require the variables `jobId` and `data` in the context of the graphql execution.


The context is the third argument passed to `graphqlObservable(doc, schema, context)`